src/descriptor/tr.rs

```diff
@@ -623,6 +683,7 @@ impl<Pk: MiniscriptKey, Ext: Extension> Liftable<Pk> for TapTree<Pk, Ext> {
                     Ok(Policy::Threshold(1, vec![lift_helper(l)?, lift_helper(r)?]))
                 }
                 TapTree::Leaf(ref leaf) => leaf.lift(),
+                TapTree::SimplicityLeaf(..) => panic!("Cannot lift Simplicity policy to Miniscript semantic policy"),
```

Comment:
In 899269acefeb534f140cfd4d7f86b66ad567b55f:

Can you stick the word FIXME into this error message?

Reply:
This is the only panic added in the first commit which isn't removed by the end of the PR.